src/solvers.jl

```diff
@@ -80,6 +85,119 @@ function addQuadratics(m::Model)
             error("Solver does not support quadratic constraints")
         end
     end
+    nothing
+end
+
+function addTransConstrs(m::Model)
+    for c in m.normconstr
+        process_constraint(m,c)
+    end
+    nothing
+end
+
+function process_constraint(m::Model, c::NormConstraint{1})
+    auxidx = m.numCols + m.numAuxCols
+    startidx = auxidx + 1
+    for term in c.normexpr.norm.terms
+        MathProgBase.addvar!(m.internalModel, 0.0, Inf, 0.0)
+        auxidx += 1
+        # aux >= aff  <-->  aux - aff.terms >= aff.constant  <--> -aux + aff.terms <= -aff.constant
+        @show (vcat(Int[v.col for v in term.vars], auxidx),
+                                vcat(term.coeffs, -1.0),
+                                -Inf,
+                                -term.constant)
+        MathProgBase.addconstr!(m.internalModel,
```

Comment:
We're implicitly requiring here that LP solvers implement ``addconstr!``. We've never required this before, and Cbc and the conic solvers don't currently support this.